index.js

```diff
@@ -55,26 +55,41 @@ const text = [
         text: "pretzels got wet",
         weight: 10
     },
+    {
+        text: "Blaseball is a mess.",
+        weight: 10
+    },
 ];
 
 const textArray = text.map(saying => Array(saying.weight).fill(saying.text)).flat();
 
 function onLoad() {
-    const hatType = (new URLSearchParams(window.location.search)).get('hat');
-    if (hatType === 'chef') {
-        const floaty = document.querySelector(".floaty");
-        if (!floaty.src.includes('squid-hat.png')) {
-            floaty.src = 'squid-hat.png';
-            floaty.alt = "A giant squid, adorned with a chef's hat, floating ominously on the page";
-            return;
+    const params = new URLSearchParams(window.location.search)
+    const type = params.get('type');
```

Comment:
For backwards compatibility (so the `?hat=chef` links work), can we make this:
```
    const params = new URLSearchParams(window.location.search);
    const type = params.get('type') || params.get('hat');
```

Reply:
Done.